apps/full-site-editing/bin/sync-newspack-blocks.sh

```diff
@@ -126,6 +126,7 @@ cp -R $CODE/src/components $TARGET/
 
 # Fix the text domain
 npx eslint . --fix > /dev/null 2>&1
+phpcbf $TARGET
```

Comment:
Usually in bash I see variables like this wrapped with `""` but I see that's not the case in lines above. 🙃